websites/V/Video Game Mods/presence.ts

```diff
@@ -0,0 +1,159 @@
+const presence = new Presence({
+		clientId: "1079522783974920212",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Other",
+			largeImageKey: "https://i.imgur.com/zvX0TaN.png",
+			startTimestamp: browsingTimestamp,
+			buttons: [
+				{
+					label: "View Page",
+					url: window.location.href,
+				},
+			],
+		},
+		urlpath = window.location.pathname.split("/");
+
+	switch (true) {
+		case !urlpath[1]:
+			presenceData.details = "Home";
+			break;
+		default:
+			if (urlpath[1].startsWith("members")) {
+				presenceData.details = `In ${
+					urlpath[3].length > 0 ? urlpath[3].replace("bp-", "") : "profile"
```

Comment:
Or just get it with a querySelector?
```suggestion
					document.querySelector('[class="data"]')?.textContent
```